randovania/layout/description_migration.py

```diff
@@ -695,6 +695,16 @@ def fix_elevator(conn: dict[str, str], old_name: str, new_name: str) -> None:
             )
 
 
+def _migrate_v35(data: dict) -> None:
+    game_modifications = data["game_modifications"]
+
+    for index, game in enumerate(game_modifications):
+        for region in game["locations"].keys():
+            for pickup in game["locations"][region].keys():
+                if "Energy Transfer Module" in game["locations"][region][pickup]:
+                    game_modifications[index]["locations"][region][pickup] = "Nothing"
```

Comment:
why not do something like this instead.
```py
for location, pickup in game["locations"][region].items():
  if etm in pickup:
    game_modifications[index]["locations"][region][pickup] = "Nothing"
```
?

Reply:
i want to say that you should also be able to do `location = "Nothing"` but you'd need to double cheeck that